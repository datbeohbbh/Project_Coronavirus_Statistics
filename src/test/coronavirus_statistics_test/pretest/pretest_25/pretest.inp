25 21
q 1 22 254
? 4 1
q 1 13 119
? 4 1
q 2 13 79
q 1 25 78
q 1 10 474
q 1 2 126
q 1 19 120
q 3 2 3
? 4 5
? 6
q 1 17 62
? 7
q 1 3 329
q 2 10 181
? 4 5
? 4 8
? 7
? 4 8
? 4 2