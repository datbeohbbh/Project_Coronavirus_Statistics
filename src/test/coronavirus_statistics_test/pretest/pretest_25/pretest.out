Query ? 4 1 :
The answer is the city with the index 22 
- Information of the city 22 -
-> Number of current patients : 254
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 13 
- Information of the city 13 -
-> Number of current patients : 119
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 22 
- Information of the city 22 -
-> Number of current patients : 254
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 13
Quantity: 79 people

Query ? 7 :
The city with the highest death is city with the index 2
Quantity: 3 people

Query ? 4 5 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 123
-> Has recovered : 0
-> Was dead : 3

Query ? 4 8 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 329
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 2
Quantity: 3 people

Query ? 4 8 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 329
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 17 
- Information of the city 17 -
-> Number of current patients : 62
-> Has recovered : 0
-> Was dead : 0

